### PR TITLE
finished code challenge 15

### PR DESCRIPTION
**Pull request by egconley**
- [x] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [NA] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/tree/TreeTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+public class TreeTest {
+
+    Tree<String> tree;
+
+    @Before
+    public void setup() {
+        tree = new Tree<String>(new Node<String>("A",
+                new Node<String>("B", new Node<String>("D"), new Node<String>("E")),
+                new Node<String>("C", new Node<String>("F"), new Node<String>(null))));
```
Comment (Zaffierce) [inline code401challenges/src/test/java/code401challenges/tree/TreeTest.java:16]: Since you've already declared your tree variable as a Tree<String> on line 10, you don't need to put String within the carats.
